### PR TITLE
Automate closing PRs with missing DCO

### PR DESCRIPTION
**Pull request by xaviarias**

### CODE REVIEW

**Comment by xaviarias** [comment]
@ljacomet would it make sense to label the `pending:pr-dco` as `pending:dco`?

It doesn't make sense outside of the context of  a PR and would be more consistent with the label `closed:missing-dco`.

**Comment by xaviarias** [comment]
Thanks for the feedback!

File: .github/workflows/stale-issue.yml [hunk 1]
```diff
@@ -111,5 +112,9 @@ jobs:
             In the meantime, closing as unreproducible.
 
           # PULL REQUESTS -----------------------------------------------------
-          stale-pr-label: 'pending:reproducer'
-          days-before-pr-close: -1
+          days-before-pr-close: 7
+          stale-pr-label: 'pending:pr-dco'
```
Comment (ljacomet) [inline .github/workflows/stale-issue.yml:116]: Let's rename the label to `pending:dco` as suggested and get this merged.

**Comment by xaviarias** [comment]
@bot-gradle test and merge

**Comment by blindpirate** [comment]
@bot-gradle test and merge

**Review by blindpirate — APPROVED** [review]

**Pull request merged**